data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -1017,6 +1080,55 @@ spec:
             description: Platform is the configuration for the specific platform upon
               which to perform the installation.
             properties:
+              alibabacloud:
+                description: AlibabaCloud is the configuration used when installing
+                  on Alibaba Cloud.
+                properties:
+                  defaultMachinePlatform:
+                    description: DefaultMachinePlatform is the default configuration
+                      used when installing on Alibaba Cloud for machine pools which
+                      do not define their own platform configuration.
+                    properties:
+                      instanceType:
+                        description: InstanceType defines the ECS instance type. eg.
+                          ecs.xn4.small
```

Comment:
refer to previous instance size recommendations.

Reply:
Ok, will changed it later

Reply:
finish updated